client/src/app/domain/models/mediafiles/mediafile.ts

```diff
@@ -38,12 +38,44 @@ export class Mediafile extends BaseModel<Mediafile> {
         super(Mediafile.COLLECTION, input);
     }
 
-    public used_as_logo_in_meeting_id(place: string): Id | null {
+    /**
+     * Check if the image is used at the given place. If place is empty, checks
+     * if the image is used in any place and returns the first meeting id found.
+     *
+     * @param place The places the image can be used
+     * @returns the meeting id or `null` if the image is not used.
+     */
+    public used_as_logo_in_meeting_id(place?: string): Id | null {
+        if (!place) {
```

Comment:
This function is extremely similar to `used_as_font_in_meeting_id`. Maybe look into creating a helper function that handles the shared functionality.

Reply:
This is better, but wouldn't it be possible to also handle the rest of these functions in `used_in_meeting`, as the old code was already basically the same?